topology/doc/make_manual.sh

```diff
@@ -42,5 +43,6 @@ ${MKIDX}  ${fname}
 ${LATEX}  ${fname}
 ${LATEX}  ${fname}
 
-# remove temporary files
+# remove temporary files and figures
 rm -f ${fname}.{aux,bbl,blg,idx,ilg,ind,ins,lof,log,lot,synctex.gz,toc}
+rm -rf user_manual_figures
```

Comment:
End the last line of the file with a newline.

Reply:
Done.